internal/files/compress.go

```diff
@@ -36,7 +36,10 @@ func Zip(sourcePath, destinationFile string) error {
 	}
 	defer os.RemoveAll(tempDir)
 	workDir := filepath.Join(tempDir, folderNameFromFileName(destinationFile))
-	os.MkdirAll(workDir, 0755)
+	err = os.MkdirAll(workDir, 0755)
+	if err != nil {
+		return errors.Wrap(err, "can't prepare work directory")
```

Comment:
nit: `errors.Wrapf( ..., workDir)`?

Reply:
That will be in the underlying `os.PathError`, but done. pkg/errors makes me sad.